### PR TITLE
dynamically change logos based on geolocation

### PR DESCRIPTION
**Pull request by NathHorrigan**

### CODE REVIEW

**Comment by NathHorrigan** [comment]
This implementation is based on the free version of the https://ipdata.co API, It's used by major companies and the free version gives us 1500 daily requests, if we go over this limit then the US logos won't switch over which isn't a drastic failure IMHO. I have the free API registered under my TBX email but anyone can easily create another for running on dev etc.

File: src/templates/service/service-page.js [hunk 1]
```diff
@@ -103,6 +104,18 @@ class ServicePage extends React.Component {
     }
   }
 
+  updateGeoRegion = async () => {
+    fetch(
+      `https://api.ipdata.co/?api-key=${process.env.GATSBY_LOCATION_API_KEY}`
```
Comment (kaedroho) [inline src/templates/service/service-page.js]: Does this API key you have, have domain whitelisting enabled?

Comment (NathHorrigan) [reply src/templates/service/service-page.js]: Sorry, I don't follow. Where would it need to be whitelisted?

**Comment by helenb** [comment]
Closing this pull request as this is going to be implemented in a different way.

**Pull request closed without merging**